src/UserStories/UserStories.css

```diff
@@ -139,64 +146,49 @@ html {
   border-radius: 22px;
   height: 44px;
   padding-left: 30px;
-  margin-bottom: 25px;
+  margin-bottom: 10px;
   box-sizing: border-box;
   align-self: flex-end;
   background: none;
   text-overflow: ellipsis;
-
   white-space: nowrap;
   overflow: hidden;
 }
 ::-webkit-input-placeholder {
   color: white;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
     Arial, sans-serif;
   max-width: 110px;
 }
 .direct-story-icon {
   align-self: flex-end;
-  margin-bottom: 35px;
+  margin-bottom: 20px;
   padding: 0 5px 0 15px;
 }
 
 #user-avatar-small {
   position: absolute;
-  top: 250%;
+  top: 280%;
   right: 45%;
-  transform: scale(2);
+  transform: scale(2.5);
 }
```

Comment:
I don't recommend this at all. You should contain all of the small story stuff in a div that is centered vertically and horizontally. I.e create a container div that is a flex. 

I see that you're scaling to counter-effect the 0.4 scale of the small story. This is not great because you're doing two things instead of nothing. Try to make the 0.4 scale only apply to the background, not the whole thing, if it's too hard, let me know and we'll figure it out together.

At least when you have one container div for everything, you can apply the scale to it only.